src/app/_components/Badge.tsx

```diff
@@ -1,12 +1,28 @@
+import clsx from 'clsx'
+
 type BadgeProps = {
-  featured: boolean
-  children?: string
+  variant?: 'primary' | 'ghost',
+  className?: string,
+  children: React.ReactNode
+}
+
+const variantStyles = {
+  primary:
+    'bg-brand-300 text-brand-800',
+  ghost:
+    'bg-transparent text-brand-100 border-current border',
```

Comment:
Let's make it `bg-brand-800` instead of `transparent` and explicitly set the border color. 🙏🏼